charts/kube-otel-stack/templates/bridge.yaml

```diff
@@ -0,0 +1,63 @@
+{{ $bridge := $.Values.opampBridge}}
+{{ if $bridge.enabled }}
+{{ $bridgeName := (print $.Release.Name "-bridge") }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpAMPBridge
+metadata:
+  name: {{ $bridgeName }}
+  {{- with $bridge.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    app: {{ $bridgeName }}
+    {{- include "kube-otel-stack.labels" $ | indent 4 }}
+spec:
+  image: {{ $bridge.image }}
+  endpoint: {{ $bridge.endpoint }}
+  {{- with $bridge.capabilities }}
+  capabilities:
+    {{-  toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with $bridge.headers }}
+  headers:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with $bridge.env }}
+  env:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

Comment:
We should make the rest of the fields configurable here

Reply:
`rest of the fields` as in everything from the [bridge spec](https://github.com/open-telemetry/opentelemetry-operator/blob/e58ce39093d7d35a0e9ba1dc30acfe19770e9fa8/apis/v1alpha1/opampbridge_types.go#L23)?

Reply:
yes. you can use whati have here :) https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1058/files#diff-b2529bea77126c97aab5b4736da2da8f4d17ab49f93c4f0d6ee5f569e0b2f6cf

Reply:
pulled in a lot of this! helpers includes :-)
can you check the names of the cluster role/cluster role binding in the PR description and see if those look acceptable? i changed some `include`s around a little bit since all of these charts are in the same file. i havent tried applying this to an environment yet but i will test that too

Reply:
you'll need to do it for this chart too :)

Reply:
i removed this chart from this PR since its just going to be the same, so theres less duplication and less chance of drift between the two while editing both. When this PR is merged ill put up a second PR adding the same stuff to kube-otel-stack

Reply:
sounds good!